src/main/java/org/springframework/cloud/dataflow/module/deployer/cloudfoundry/ApplicationModuleDeployer.java

```diff
@@ -163,6 +164,23 @@ public void run() {
 		return moduleDeploymentId;
 	}
 
+	private int deduceMemory(Map<String, String> deploymentProperties) {
+		String override = deploymentProperties.get("cloudfoundry.memory");
```

Comment:
Actually, using the manifest format wouldn't work, since the API requires a numeric value.